ocrd_network/ocrd_network/process_helpers.py

```diff
@@ -29,10 +41,11 @@ def invoke_processor(
                 output_file_grp=output_file_grps_str,
                 page_id=page_id,
                 parameter=parameters,
-                instance_caching=True
+                instance_caching=True,
+                mets_server_url=mets_server_url
             )
         except Exception as e:
-            raise RuntimeError(f"Python executable '{executable}' exited with: {e}")
+            raise RuntimeError(f"Python executable '{processor_class.__dict__}' exited with: {e}")
```

Comment:
Why that change? If for debugging, perhaps extend to include both executable and the class dict?

Reply:
It was just overlooked in the previous implementation. `{executable}` for `run_cli`, but `processor_class.__dict__` for `run_processor`. It was just `{executable}` for both and always resolved to None in the `run_processor` case.